entproto/cmd/entproto/helpers.txt

```diff
@@ -0,0 +1,21 @@
+package entpb
+
+func toRepeatedInt(repeatedInt64 []int64) []int {
```

Comment:
the way to add more code to the generated code is by extending the templates found here:
https://github.com/ent/contrib/blob/master/entproto/cmd/protoc-gen-entgrpc/template/to_ent.tmpl
https://github.com/ent/contrib/blob/master/entproto/cmd/protoc-gen-entgrpc/template/to_proto.tmpl